Modules/mathmodule.c

```diff
@@ -2016,6 +2016,71 @@ math_factorial(PyObject *module, PyObject *arg)
 }
 
 
+/*[clinic input]
+math.lcm
+
+    x as a: object
+    y as b: object
+    /
+least common multiple of x and y
+[clinic start generated code]*/
+
+static PyObject *
+math_lcm_impl(PyObject *module, PyObject *a, PyObject *b)
+/*[clinic end generated code: output=6f83fb6d671074ba input=bd41b785dc2a4ff1]*/
+{
+    PyObject *g, *m, *f, *ab;
+    
+    a = PyNumber_Index(a);
+        if (a == NULL) {
+            return NULL;
```

Comment:
We execute this line if the `PyNumber_Index` call above failed. In this case, an error message will already have been set and we're responsible for DECREF'ing all owned references and then returning `NULL` to the caller to pass on the failure indication. But at this point, we don't own any references yet, so there's nothing to DECREF and we can just return `NULL`.

Owned references before this line: None